service-workers/service-worker/resources/clients-get-worker.js

```diff
@@ -4,12 +4,10 @@
 // the `event` object. In the case of the `onmessage` handler, it provides the
 // Client instance attributes of the requested clients.
 self.onfetch = function(e) {
-  if (e.request.mode === 'navigate' && e.clientId !== "") {
-    e.respondWith(Response.error(
-      '`clientId` incorrectly set to non-empty string for request with mode `navigate`'
-    ));
-    return;
-  }
+  // NOTE: Until FetchEvent.resultingClientId is implemented some browsers
+  // return null for the FetchEvent.clientId on navigation requests.  Since
+  // this is not in the spec yet, though, we don't strictly enforce it in the
+  // test any more.
```

Comment:
Yes I think we should use fetch-event.https.html to test this instead (although the current test there is a bit awkward).

See also client-navigate-worker.js which seems to explicitly test the opposite.

I don't know if we really need this comment in this file though... seems better suited for the commit description.